tests/model_selection/test_keras.py

```diff
@@ -0,0 +1,81 @@
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+import pytest
+from distributed import Nanny
+from distributed.utils_test import gen_cluster
+from packaging import version
+from scipy.stats import loguniform
+
+from dask_ml.model_selection import IncrementalSearchCV
+
+try:
+    import tensorflow as tf
+    from tensorflow.keras.datasets import mnist as keras_mnist
+    from tensorflow.keras.layers import Dense
+    from tensorflow.keras.models import Sequential
+    import scikeras
+    from scikeras.wrappers import KerasClassifier
+
+    pytestmark = [
+        pytest.mark.skipif(
+            version.parse(tf.__version__) < version.parse("2.3.0"),
+            reason="pickle support",
+        ),
+        pytest.mark.skipif(
+            version.parse(scikeras.__version__) < version.parse("0.1.8"),
+            reason="partial_fit support",
+        ),
+    ]
+except ImportError:
+    pytestmark = pytest.mark.skip(reason="Missing tensorflow or scikeras")
+
+
+def mnist() -> Tuple[np.ndarray, np.ndarray]:
+    (X_train, y_train), _ = keras_mnist.load_data()
```

Comment:
Maybe we shouldn't download mnist as part of the test suite.  Maybe there is some smaller testing dataset that we can play with instead?

Reply:
Thanks!

Reply:
Now, I mirror the MNIST dataset.